lib/instance-detail-helper.js

```diff
@@ -132,6 +133,24 @@ function getGenuineMongoDB(results, done) {
   done(null, res);
 }
 
+function getDataLake(results, done) {
+  const buildInfo = results.build.raw;
+
+  debug('isDataLake check: buildInfo.queryEngine', buildInfo.queryEngine);
```

Comment:
double that you want to keep this one here! 

(i know we had this conversation before, but it was in a different instance so i am not sure if applies to this as well)

Reply:
This is on purpose! In case something is not showing up, it's good to see :)